### DISCUSSION TITLE
HeaderFilter to support other filters like FilterRow

### DISCUSSION THREAD

**Discussion by Ofear** [komarovalexander/ka-table#187] [Ideas]
Hi, Great lib.

I'm probably will add it to our project.

One thing that I need is to show the user a header filter(excel-like) with filterRow depending on the content.

For example, when the DataType is Date, clicking on the date will open a date picker with range or with conditional options.
Or for example when a DataType is a number then it will have input with greater or smaller etc,

Currently, the header filter supports only the checkbox.

It will be great addition to allow FilterRow inside the HeaderFilter 

And for a future feature, is to support a total excel-like header filter with search filter value and select all\none

For example:
![image](https://user-images.githubusercontent.com/7635725/144737850-8b6e6624-f064-434d-99bf-bd90f9d24607.png)

**Comment by komarovalexander** [comment]
the same question #207

**Comment by komarovalexander** [comment]
Hi @Ofear in v11.0.0 search has been added to the HeaderFilter https://komarovalexander.github.io/ka-table/#/header-filter-logic
